Sources/SSChart/Bar/Stacked/StackedBarChart.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  StackedBarChart.swift
+//  
+//
+//  Created by YJ on 2022/09/02.
+//
+
+import Foundation
+import UIKit
+
+public class StackedBarChart: UIView, Chart {
+    
+    // MARK: - public
+    public var items: [StackedBarChartItem] = [] {
+        didSet {
+            setNeedsLayout()
+            layoutIfNeeded()
+        }
+    }
+    
+    // MARK: - private
+    private var percentages: [ChartItemValuePercentage] = []
+    private var bars: [UIView] = []
+    
+    private let animationDelay: Double = 0.3
+    private var didAnimation: Bool = false
+    private var maxBarWidth: CGFloat = 0
+    
+    // MARK: user custom
+    private let itemSpacing: Double
+    private let itemCornerRadius: Double
+    private let showPercentage: Bool
+    private let percentageLabelFont: UIFont
+    private let percentageLabelTextColor: UIColor
+    private let animationDuration: Double
+    private let barMoveUpSpace: Double
+    private let isAnimationPaused: Bool
+    
+    // MARK: - init
+    /// - Parameters:
+    ///   - frame: frame of chart
+    ///   - itemSpacing: space between item(bar). Default 0.0
+    ///   - itemCornerRadius: corner radius of item. Default 0.0
+    ///   - showPercentage: Bool determines to show percentage label. Default true
+    ///   - percentageLabelFont: font of percentage label. Default systemfont of size 12
+    ///   - percentageLabelTextColor: text color of percentage label. Default white
+    ///   - animationDuration: animation duartion. Default 0.8
+    ///   - barMoveUpSpace: space that bar moves up during animation. Default 20
+    ///   - isAnimationPaused: bool indicates to pause animation at the beginning. Default false
+    public init(frame: CGRect, itemSpacing: Double = 0.0, itemCornerRadius: Double = 0.0, showPercentage: Bool = true, percentageLabelFont: UIFont = UIFont.systemFont(ofSize: 12), percentageLabelTextColor: UIColor = UIColor.white, animationDuration: Double = 0.5, barMoveUpSpace: Double = 0, isAnimationPaused: Bool = false) {
+        self.itemSpacing = itemSpacing
+        self.itemCornerRadius = itemCornerRadius
+        self.showPercentage = showPercentage
+        self.percentageLabelFont = percentageLabelFont
+        self.percentageLabelTextColor = percentageLabelTextColor
+        self.animationDuration = animationDuration
+        self.barMoveUpSpace = barMoveUpSpace
+        self.isAnimationPaused = isAnimationPaused
+        
+        super.init(frame: frame)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - override
+    public override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        reload()
+        
+        if isAnimationPaused {
+            pauseAnimation()
+        }
```

Comment:
Only StackedBarChart calls `if isAnimationPaused { pauseAnimation() }` in `layoutSubviews` while others call `pauseAnimation()` in `reload()` function. Consider calling `pauseAnimation()` method from `reload()` method in `Chart` protocol extension.